Dockerfile

```diff
@@ -41,7 +42,7 @@ RUN mkdir /app/tmp
 COPY ./requirements.txt /app/tmp
 RUN pip install --no-cache-dir -r /app/tmp/requirements.txt
 COPY . /app/tmp
-RUN pip install /app/tmp && rm -fr /app/tmp
+RUN pip install --no-cache-dir /app/tmp && rm -fr /app/tmp
```

Comment:
And to shrink even more this last part could be simplified into

```Dockerfile
COPY . /app/src/oca-github-bot
RUN pip install --no-cache-dir -r /app/src/oca-github-bot/requirements.txt -e /app/src/oca-github-bot
```

so we get rid of `/app/tmp` which did hang around in layers.

Reply:
Do you want to remove the src (source code directory) after the installation?

Reply:
> Do you want to remove the src (source code directory) after the installation?

No because if we install it in editable mode (-e), it is the only place where the code is available, so it is important to keep it.

Reply:
I changed the file with your suggestion above and did a built, but the only size that was reduces, where the number of code lines :)

Reply:
my bad, I confused it with code you have to compile (make install), but python is an interpreted language